content/blogposts/2022/release-3.41.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Sourcegraph 3.41 release"
+publishDate: 2022-06-21T10:00-07:00
+description: "Sourcegraph 3.41 introduces..."
+tags: [blog, release]
+slug: "release/3.41"
+published: false
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/release-post/3.41/sourcegraph-3-41-release.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/release-post/3.41/sourcegraph-3-41-release.png
+changelogItems: 
+  - description: 'A new templating variable, batch_change_link has been added for more control over where the "Created by Sourcegraph batch change ..." message appears in the published changeset description.'
+    url: https://github.com/sourcegraph/sourcegraph/pull/35319
+    category: Batch Chanages
+  - description: Sort and limit filters have been added to capture group insights. This gives users more control over which series are displayed.
+    url: https://github.com/sourcegraph/sourcegraph/pull/34611
+    category: Code Insights
+  - description: 'Commit and diff search: The hard limit of 50 repositories has been removed, so you can now run broader searches. Long-running searches will continue running until the timeout is hit.'
+    url: https://github.com/sourcegraph/sourcegraph/pull/36486
+    category: Search
+  - description: 'Zoekt-indexserver has a new debug landing page, /debug, which exposes information about the queue, the list of indexed repositories, and the list of assigned repositories for easier visual debugging. Admins can reach the debug landing page by selecting Instrumentation > indexed-search-indexer from the site admin view. The debug page is linked at the top.'
+    url: https://github.com/sourcegraph/zoekt/pull/346
+    category: Admin
+---
+ 
+Sourcegraph 3.41 is now available! Here are some highlights from this release:
+
+
+## Batch Changes
+
+### Running batch changes server-side is moving to Beta
```

Comment:
```suggestion
### Server-side batch changes are now available in Beta
```

@malomarrec, when you say "server-side batch changes", do you always keep "batch changes" lowercase?
Also @lcolston08 for your vis!

Reply:
@iskyOS the name that we agreed upon previously was to drop server-side batch changes, and to use "running batch changes server-side"
https://docs.google.com/document/d/1AhuTd7NtzaBLu3CBpr-lr7JO_0Sc-yj6kTRW_9j8RYU/edit#heading=h.j70r4vtdxyp1
This is to avoid it being treated as a "product"

Reply:
Thanks for the context!